modules/widgets/twitter-timeline.php

```diff
@@ -82,6 +82,16 @@ public function admin_scripts( $hook ) {
 	 * @param array $instance Saved values from database.
 	 */
 	public function widget( $args, $instance ) {
+		if ( $this->is_widget_id_deprecated( $instance['type'] ) ) {
+			if ( current_user_can( 'edit_theme_options' ) ) {
+				echo $args['before_widget'];
+				echo $args['before_title'] . esc_html__( 'Twitter Timeline', 'jetpack' ) . $args['after_title'];
+				echo '<p>' . esc_html__( "The Twitter Timeline widget can't display tweets based on searches or hashtags. To display a simple list of tweets instead, change the Widget ID to a Twitter username. Otherwise, delete this widget.", 'jetpack' ) . '</p>';
```

Comment:
No need for double quotes, single quotes can be used.

Reply:
There's a single quote `widget can't display` in the string that I'd rather not escape 🙂